### ISSUE TITLE
API tester can not send post or get to another domain

### ISSUE THREAD

**Issue by herb123456** [Codeception/Codeception#2149]
this is my config
```
class_name: ApiTester
modules:
    enabled:
        - \Helper\Api
        - PhpBrowser:
            url: http://localhost:9000
            proxy: localhost:8081
        - REST:
            url: http://localhost:9000
            depends: PhpBrowser
```

and my test code

```

        $I->sendGET("http://google.com");

        $I->sendPOST("http://localhost:9000/test");
```

and the result is

```
I send get "http://google.com"
  [Request headers] []
  [Request] GET http://google.com
  [Response]
  [Headers] {"Host":["localhost:9000"],"Connection":["close"],"X-Powered-By":["PHP/5.6.8"],"Content-type":["text/html; charset=UTF-8"],"Content-Type":["text/html"]}
  [Status] 200
  I send post "http://localhost:9000/test"
  [Request headers] []
  [Request] POST http://localhost:9000/test []
  [Response] aaa
  [Headers] {"Host":["localhost:9000"],"Connection":["close"],"X-Powered-By":["PHP/5.6.8"],"Content-type":["text/html; charset=UTF-8"],"Content-Type":["text/html"]}
  [Status] 200
```

and the proxy is my debug tool

I can see every request domain are localhost:9000

and the domain is follow my config

but it does not have this issue in 1.8.7 version

**Comment by DavertMik** [comment]
Please try Codeception 2.1.1

**Comment by Naktibalda** [comment]
This issue has been fixed in 2.1.1 release.

**Comment by umang-ranium** [comment]
@DavertMik Is there a way to test javascript errors on page
